qiskit_machine_learning/algorithms/classifiers/neural_network_classifier.py

```diff
@@ -75,8 +77,12 @@ def __init__(
         """
         super().__init__(neural_network, loss, optimizer, warm_start, initial_point)
         self._one_hot = one_hot
+        # encodes the target data if categorical
+        self._target_encoder: Optional[Union[OneHotEncoder, LabelEncoder]] = None 
```

Comment:
```suggestion
        # encodes the target data if categorical
        self._target_encoder: Optional[Union[OneHotEncoder, LabelEncoder]] = None
```
Removed trailing white space - third time lucky!